app/src/main/java/com/abbyberkers/apphome/ns/XMLprocessing.kt

```diff
@@ -0,0 +1,14 @@
+package com.abbyberkers.apphome.ns
+
+import com.abbyberkers.apphome.converters.delayToInt
+import com.abbyberkers.apphome.converters.toDelayString
+import kotlin.math.max
+
+fun maximumDelay(delay: String?, otherDelay: String?): String? {
```

Comment:
Please check if all your functions have documentation. In particular this one I noticed, I will not know how to call it without documentation: what to expect when I put in a null value? Why is it allowed putting in null values?
Next time, try this order:
- Write documentation of the function (it will help actually implementing it)
- Write tests for the functions, probably most of them fail
- Implement the function